thecjharries/advent-of-code-2023
# feat/day 14

- Add day 14 boilerplate
- Create and test transpose
- Implement and test map parsing
- Implement and test tilting
- Test part 1
- Implement part 1
- Implement and test fnc to rotate matrix
- Test part 2
- Ensure rotation returns to original
- Flip from clockwise to counterclockwise
- Implement and test fnc to cycle dish
- Implement part 2
- Remove print_map
- Remove unnecessary mut
